runtime/compiler/env/J9VMMethodEnv.hpp

```diff
@@ -54,6 +54,30 @@ class OMR_EXTENSIBLE VMMethodEnv : public OMR::VMMethodEnvConnector
 
    uint32_t bytecodeSize(TR_OpaqueMethodBlock *method);
 
+   /**
+    * @brief Given a method signature, tokenize it into its class name, method name,
+    *        and type signature strings.
+    *
```

Comment:
It would be nice to document what sort of pattern(s) it's looking for, e.g. in regex form or an example signature or something.